README.md

```diff
@@ -8,17 +8,15 @@ Works on most Debian and CentOS/RHEL based distributions!
 1.  Install Raspbian
 2.  Run the command below (downloads [this script](https://github.com/pi-hole/pi-hole/blob/master/automated%20install/basic-install.sh) in case you want to read over it first!)
 
-## `curl -L https://install.pi-hole.net | bash`
+### `curl -sSL https://install.pi-hole.net | bash`
 
 ### Alternative Semi-Automated install
 
 ```bash
 wget -O basic-install.sh https://install.pi-hole.net
-chmod +x basic-install.sh
-./basic-install.sh
+cat basic-install.sh | bash
```

Comment:
`bash basic-install.sh` to get the same effect.  useless cat / pipe, reduces complexity.

[shellcheck agrees](https://github.com/koalaman/shellcheck/blob/619b6c42f3f53fd6d0aea5ae282b8026c403a19a/ShellCheck/Analytics.hs#L354)